AdobeStockImageAdminUi/Test/Mftf/Section/AdobeStockSection.xml

```diff
@@ -19,9 +19,10 @@
         <element name="gridSpinner" type="text" selector="//div[contains(@id, 'adobe-stock-images-search-modal')]//div[@class='spinner']"/>
         <element name="modalError" type="text" selector="//aside[contains(@class, 'modal-popup')]//div[@class='modal-content']//div[text()='Something went wrong.']"/>
         <element name="countGridImages" type="text" selector="//div[@class='masonry-image-grid']/div[contains(.,data-repeat-index)]"/>
-        <element name="recordsFound" type="text" selector="//*[@id='adobe-stock-images-search-modal']//*[text()='records found']/parent::div"/>
         <element name="savePreview" type="block" selector="//button[@class='action-secondary']/span[contains(.,'Save Preview')]"/>
         <element name="imagePreview" type="block" selector="//div[@class='masonry-image-preview']//img"/>
+        <element name="recordsFound" type="text" selector="//*[@id='adobe-stock-images-search-modal']//*[text()='records found']/parent::div"/>
         <element name="imagePreviewNavigation" type="block" selector="//div[@class='masonry-image-preview']//div[@class='action-buttons']/button[@class='action-{{type}}']" parameterized="true"/>
+        <element name="selectedImagePreview" type="text" selector="//small/parent::*[@class='filecnt selected']//img"/>
```

Comment:
This selector doesn't seem to be reliable? Should the item be placed in a specific container? Why are we looking for a parent of a "small" element?

Reply:
If you have better selector for selected image, please share, because i tried a lot of selectors only this worked for me.
![DeepinScreenshot_select-area_20190827123246](https://user-images.githubusercontent.com/51681547/63759940-0d60b700-c8c7-11e9-8d41-329c24b975fb.png)

Reply:
@engcom-Golf please adjust the code to provide reliable selector